csharp/Platform.Collections/Lists/IListExtensions.cs

```diff
@@ -12,9 +12,9 @@ public static class IListExtensions
         /// <para>Gets the element from specified index if the list is not null and the index is within the list's boundaries, otherwise it returns default value of type T.</para>
         /// <para>Получает элемент из указанного индекса, если список не является null и индекс находится в границах списка, в противном случае он возвращает значение по умолчанию типа T.</para>
         /// </summary>
+        /// <typeparam name="T"><para>The list's item type.</para><para>Тип элементов списка.</typeparam>
         /// <param name="list"><para>The checked list.</para><para>Проверяемый список.</para></param>
         /// <param name="index"><para>The index of element.</para><para>Индекс элемента.</para></param>
-        /// <typeparam name="T"><para>The list's item type.</para><para>Тип элементов списка.</typeparam>
         /// <returns>
         /// <para>If the list is not null and the index is within list's boundaries.</para>
```

Comment:
`If the specified index is within list's boundaries, then - list[index], otherwise the default value.`